marlowe-playground-client/static/css/projects.css

```diff
@@ -0,0 +1,39 @@
+.projects-container {
+  width: 90%;
+}
+
+.projects-container h1 {
+  margin: 1.5rem 0rem;
+}
+
+.projects-container hr {
+  margin-bottom: 2rem;
+}
+
+.gists tr:nth-child(even) {
```

Comment:
I don't like the styling for this table, and I know I've already said it on a PR-level comment, but it's going in the review too. 😛

Reply:
for another PR